internal/nets/nets.go

```diff
@@ -87,25 +70,55 @@ func List(offset, limit int, mls paramstores.NetParamStore) ([]types.BriefNet, i
 	return nets, cursor, nil
 }
 
-// Required returns the number of patterns required to train a net with the specified topology
-func Required(inputs, outputs, hLayers int, conf config.Config) (int, error) {
-	switch conf.ML.Net {
-	case "mlp":
-		topology := MLPTopology(inputs, outputs, hLayers)
-		w := 0
-		for layer, neurons := range topology[:len(topology)-1] {
-			w += (neurons + 1) * topology[layer+1]
+// LoadNetwork is used to retrieve network params for a given ID and build a network object with them. When they can't
+// be found (nil, nil) will be returned
+func LoadNetwork(id, nType string, nps paramstores.NetParamStore) (Network, error) {
+	switch nType {
+	case types.MultilayerPerceptron:
+		var np paramstores.MLPParams
+		found, err := nps.Load(id, &np)
+		if err != nil {
+			return nil, err
+		}
+		if !found {
+			return nil, nil
 		}
-		return int(math.Ceil(float64(w) / 0.1)), nil
+		return MLPFromParams(id, np)
 	default:
-		return 0, errors.New(conf.ML.Net + " is not a valid net type")
+		return nil, errors.New(nType + " is not a valid net type")
 	}
 }
 
+// Required returns the number of patterns required to train a net (of the most demanding type) with the specified
+// topology
+func Required(inputs, outputs, hLayers int, conf config.Config) int {
+	// required := []int{}
+
+	// MLP
+	topology := MLPTopology(inputs, outputs, hLayers)
+	w := 0
+	for layer, neurons := range topology[:len(topology)-1] {
+		w += (neurons + 1) * topology[layer+1]
+	}
+	/*
```

Comment:
Why this part is commented?

Reply:
This function is used to know when a series has enough points to be trained and it uses a formula that is specific to the only type of network that is currently supported by nerd. Since this update introduces an algorithm that will test different configs which will mean that the net type will not always be the same (when we add more), I thought it would be good to try to prepare it and have it calculate the number of patterns for each type to return the maximum. Then I realized that for now, it doesn't make sense to be evaluating all these statements given that we already know it has to be an MLP. So I commented the code so we can have a bit more performance now and then uncomment and use it, when the time comes